indexing-service/src/main/java/io/druid/indexing/overlord/setup/EqualDistributionWorkerSelectStrategy.java

```diff
@@ -47,7 +47,13 @@ public int compare(
               ImmutableZkWorker zkWorker, ImmutableZkWorker zkWorker2
           )
           {
-            return -Ints.compare(zkWorker2.getCurrCapacityUsed(), zkWorker.getCurrCapacityUsed());
+            int comparation = -Ints.compare(zkWorker2.getCurrCapacityUsed(), zkWorker.getCurrCapacityUsed());
+
+            if(comparation == 0) {
+              comparation = zkWorker2.getWorker().getIp().compareTo(zkWorker.getWorker().getIp());
```

Comment:
Both solutions works too. Other posible is compare second comparation using version or hostname. Yeah the solution on `FillCapacityWorkerSelectStrategy` is similar that i did. I'm going to change IP Address to Host to do two comparator more similar.